backend/users-service/app/src/index.ts

```diff
@@ -1,7 +1,72 @@
-import { Elysia } from "elysia";
+import { Elysia, t } from 'elysia'
+import { swagger } from '@elysiajs/swagger'
+import { cors } from '@elysiajs/cors'
+import { helmet } from 'elysia-helmet'
+import jwt from '@elysiajs/jwt'
+import { PrismaClient } from '@prisma/client'
+import { logger } from '@bogeychan/elysia-logger'
 
-const app = new Elysia().get("/", () => "Users service").listen(80);
+const db = new PrismaClient()
+
+const app = new Elysia({ prefix: '/users' })
+  .use(cors())
+  .use(helmet())
+  .use(
+    jwt({
+      name: 'jwt',
+      secret: process.env.JWT_SECRET as string
+    })
+  )
+  .use(logger())
+  .use(swagger())
+  .model({
```

Comment:
i'm not sure how Elysia works for this also, but is there a way to avoid chaining like this? because its not practical in the long run and its already quite cumbersome to read

Reply:
Refactored in 89f565cb95ba774a0b66510f252952527116cfae